src/Core/Renderer/DefaultPerceivedRoomRenderer.cs

```diff
@@ -31,22 +31,42 @@ public override string Render(Thing viewer, Thing room)
             //   var exits = senses.PerceiveExits();        and also render closable exits like doors nicely; "(closed)"?
             //   var entities = senses.PerceiveEntities();  and also render players nicely; "(AFK)" etc?
             //   var items = senses.PerceiveItems();        and also track tick or build sense-specific strings (like hearing only while blind...)
+
             bool hasNoticedSomething = false;
+
+            // Handle exits differently from other Thing types
+            var exits = senses.PerceiveExits();        // TODO: Aso render closable exits like doors nicely; "(closed)"?
+            if (exits.Count > 0)
+            {
+                sb.Append($"  routes: ");
+                foreach (var exit in exits)
+                {
+                    sb.Append($"<%magenta%>{exit}<%n%>, ");
+                    hasNoticedSomething = true;
```

Comment:
I think we don't want to use the `hasNoticedSomething` flag - the purpose of that seems to be to decide whether or not to add "Here you notice: " after finding a perceivable Thing, which we probably still want to occur even if we listed one or more Routes in the line of output above it.